crosswalks/BibTeX.csv

```diff
@@ -0,0 +1,74 @@
+Property,BibTeX (@softwareversion)
+codeRepository,repository
+programmingLanguage,
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,file
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version (1)
+storageRequirements,
+supportingData,
+author,author
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,date (2) (year) / (month)
+dateModified,date (3) (year) / (month)
+datePublished,date (1) (year) / (month)

```

Comment:
https://www.bibtex.com/f/year-field/ says:

> The year field is used to store the year the work was published or in the case of an unpublished article the year it was written.

schema.org/codemeta has no concept of "the date it was written", so in my opinion, this should only map to `datePublished`, not `dateCreated` or `dateModified`.

Reply:
I got his mapping from [the codemeta to bibtex algorithm](https://gitlab.softwareheritage.org/swh/devel/swh-indexer/-/blob/master/swh/indexer/bibtex.py?ref_type=heads#L86). Do you think I should update the algorithm as well?

Reply:
It's fine for swh-indexer to differ from the crosswalks, because crosswalks can't represent anywhere near as much complexity as actual code.

Reply:
Ok!